apps/omg/test/omg/state/transaction/recovered_test.exs

```diff
@@ -114,14 +114,24 @@ defmodule OMG.State.Transaction.RecoveredTest do
       assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(<<0x80>>)
       assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(<<>>)
       assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(ExRLP.encode(23))
-      assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(ExRLP.encode([sigs, []]))
+      assert {:error, :malformed_transaction} = Transaction.Recovered.recover_from(ExRLP.encode([sigs, 1]))
```

Comment:
this `recovered_test.exs` got a little bit messy, but leaving any cleanups for a separate PR to stay sane